src/servers/grpc_server.cc

```diff
@@ -340,14 +338,17 @@ class Handler : public GRPCServer::HandlerBase {
 
   void StateRelease(State* state)
   {
-    std::unique_lock<std::mutex> lock(alloc_mu_);
+    if (max_state_bucket_count_ > 0) {
+      std::unique_lock<std::mutex> lock(alloc_mu_);
```

Comment:
Can just use `lock_guard` for scoped lock

Reply:
Fixed